### DISCUSSION TITLE
Welcome to nft-market Discussions!

### DISCUSSION THREAD

**Discussion by silviopaganini** [silviopaganini/nft-market#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by durbonca** [comment]
Hey! My name is Manuel and I really interested to learn and create NFT market. I have some experience on front with react, hoping to contribute with this project.

**Comment by durbonca** [comment]
I have this problem when I run 'Yarn Start'
React is up, but can't find exactly what could it be, any idea?

```
TypeError: Failed to construct 'URL': Invalid URL

new MiniRpcProvider
  src/index.ts:32
      29 |   ThemeUICSSObject,
      30 |   ThemeUICSSProperties,
      31 |   ThemeUIExtendedCSSProperties,
    > 32 |   ThemeUIStyleObject,
      33 |   VariantProperty,
      34 | } from '@theme-ui/css'
```

**Reply by silviopaganini** [reply]
Have you got your rpc up?

**Reply by durbonca** [reply]
am I trying with rinkeby, but I like to try with development, just I have not very clear about the WALLET_PROVIDER_URL

**Reply by silviopaganini** [reply]
I've added the .env file example in the readme

**Reply by durbonca** [reply]
thanks it help me to build the contracts

**Comment by the-one-ring** [comment]
Hi, I'm really excited to try out this project, but is it possible to add a few more setup instructions? I understand it's dependent on @silviopaganini/nft-contracts and @silviopaganini/nft-market-service but how do we tie them all together?

I would like to deploy my own NFT contract. I'd be using BSC but expect I can just use different RPC urls.

**Reply by durbonca** [reply]
Can you please share your issue with the deployment? Just need to run both (contracts and market front) actually everything is very well explained on the README file.

**Reply by the-one-ring** [reply]
@durbonca Thanks for the reply -- I think I'm missing something because I see almost no documentation in the README, the only thing I see is a list of features and a sample .env file.

I was able to get this running locally very easily, I didn't try deploying it yet. But I'd like to use my own NFT contract, and I thought there may be configuration options to specify the contract, but looks like it's dependent on the contract called NFTT in the nft-contracts repo which was built by truffle, and I already deployed my contract with https://github.com/gochain/web3 instead  -- I can do some more digging, looks like maybe I can edit `src/actions/getContract.ts` to reference my contract's ABI.

I understand this is a prototype and not a library or template. Props to the developer for the work here, it's awesome!
